lib/WP_Auth0_LoginManager.php

```diff
@@ -587,6 +576,68 @@ public function end_session() {
 		}
 	}
 
+	/**
+	 * Get and filter the scope used for access and ID tokens
+	 *
+	 * @param string $context - how are the scopes being used?
+	 *
+	 * @return string
+	 */
+	public static function get_userinfo_scope( $context = '' ) {
+		return apply_filters( 'auth0_auth_token_scope', 'openid email name nickname picture', $context );
+	}
+
+	/**
+	 * Get a base authorize URL for handling HLP redirects.
+	 *
+	 * @param null|string $connection - specific connection to use, pass null to use all enabled.
+	 * @param null|string $redirect_to - URL to redirect upon successful authentication.
```

Comment:
shouldn't this be required? where is the result taken to if not?

Reply:
`$options->get( 'default_login_redirection' );`